### PR TITLE
PR: Improvements to Conda-based Linux installer

### LINKED ISSUE #20176: Improvements to the Linux installer
I installed using the bash file. Since I did that it's not showing as installed anywhere. So, how do I uninstall it?

### PR DESCRIPTION
**Pull request by mrclary**
## Description of Changes

- Update shortcut icon for Linux
- Add post-install script for Linux: update shortcut name to allow pinning to Dash
- Updates to `napari/label/bundle_tools_2` also gave executable permissions to the installer shell script
- Alias added to `.bashrc` for command line Spyder launch in all Linux flavors
- Constructor 4.0b7 required changing post-install script substituted variables
- SPY_COMMIT SPY_BRANCH variables added to Spyder runtime environment (all platforms)
- Post-install script for macOS now unnecessary

Double-clicking the installer script should run the script in a new Terminal window in interactive mode.
To see additional installation options, execute in a Terminal:
```
$ EXPERIMENTAL-Spyder-<version>-Linux-x86_64.sh -h
```

### Issue(s) Resolved

Fixes #20176

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @mrclary

### CODE REVIEW

File: installers-conda/resources/post-install.sh [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+rm -rf ${PREFIX}
+EOF
+chmod +x ${PREFIX}/uninstall.sh
+
+cat <<EOF
+
+###############################################################################
+Spyder can be launched by standard methods in Gnome and KDE desktop
+environments. Additionally, Spyder can be launched in Gnome desktop
+environments from the command line:
```
Comment (ccordoba12) [inline installers-conda/resources/post-install.sh:48]:
```suggestion
environments. Additionally, Spyder can be launched in Gtk-based desktop
environments (e.g. Xfce) from the command line:
```

File: installers-conda/resources/post-install.sh [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+
+$ gtk-launch spyder
+
+Spyder can be launched from the command line for all Linux variants
```
Comment (ccordoba12) [inline installers-conda/resources/post-install.sh:52]:
```suggestion
Spyder can also be launched from the command line for all Linux variants
```

File: installers-conda/resources/post-install.sh [hunk 3]
```diff
@@ -1,37 +1,65 @@
...
+by:
+
+$ spyder
+
+To uninstall Spyder, from the command line:
```
Comment (ccordoba12) [inline installers-conda/resources/post-install.sh:57]:
```suggestion
To uninstall Spyder, you need to run from the following from the command line:
```

File: installers-conda/resources/post-install.sh [hunk 4]
```diff
@@ -1,37 +1,65 @@
...
+
+$ ${PREFIX}/uninstall.sh
```
Comment (ccordoba12) [inline installers-conda/resources/post-install.sh:59]: I think it'd be nice to add an `uninstall-spyder` alias to bashrc for this command and list it here instead. That would avoid people to remember what's the exact command to do it (which seems not so easy).

**Review by ccordoba12 — COMMENTED** [review]
Some small suggestions and comments for you @mrclary, otherwise looks good to me.

File: installers-conda/resources/post-install.sh (hunk 4 above)
Comment (ccordoba12) [reply installers-conda/resources/post-install.sh:59]: Uninstallation should also remove the Spyder aliases in bashrc, but that could be left for later.

**Comment by ccordoba12** [comment]
@mrclary, thanks a lot for the last improvements. This looks ready to me, but I don't have time to manually test it. Did you do that? And could you, if you didn't?

**Comment by mrclary** [comment]
> @mrclary, thanks a lot for the last improvements. This looks ready to me, but I don't have time to manually test it. Did you do that? And could you, if you didn't?

Yes, I manually tested the the full package installer, the Spyder alias, and the uninstall alias on my Ubuntu VM. When we do the release candidate for this PR, I'll test all platforms manually again as well.

**Review by ccordoba12 — APPROVED** [review]
Great work @mrclary! These are very nice improvements!

**Pull request merged**